Bottom: a81ef4f964a2493200c270667aa2b4cc0b381856
Top:    9e43b6e8c49ff2116cf429c276091dc200980943
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-01-03 12:51:46 +0000

hid/hidint.h: Remove header guard check, it appears not to be required


---

diff --git a/src/hid/hidint.h b/src/hid/hidint.h
index aa19ad5..b988db9 100644
--- a/src/hid/hidint.h
+++ b/src/hid/hidint.h
@@ -62,12 +62,8 @@ enum File_Name_Style {
 /* Returns a filename base that can be used to output the layer.  */
 const char *layer_type_to_file_name (int idx, int style);
 
-#ifdef PCB_GLOBAL_H
-
 /* Convenience function that calls the expose callback for the item,
    and returns the extents of what was drawn.  */
 BoxType *hid_get_extents (void *item);
 
-#endif
-
 void derive_default_filename(const char *pcbfile, HID_Attribute *filename_attrib, const char *suffix, char **memory);
